### PR TITLE
Implement access tracking for containingUrl

### PR DESCRIPTION
**Pull request by ntkme**
https://github.com/sass/dart-sass/issues/2208

### CODE REVIEW

File: spec/embedded_sass.proto [hunk 1]
```diff
@@ -192,6 +192,9 @@ message InboundMessage {
       // `null` instead.
       string error = 3;
     }
+
+    // Whether `containing_url` in `CanonicalizeRequest` is unused.
+    bool containing_url_unused = 4;
```
Comment (ntkme) [inline spec/embedded_sass.proto:203]: In protobuf, `bool` default to `false` when it's unset.

The default `unused = false` is effectively `accessed = true`, which means `cacheable = false`. This is the ideal default, that it works correctly even if the host does not implement the tracking, making the protocol backward compatible.

Comment (nex3) [reply spec/embedded_sass.proto:203]: Can you mention this reasoning in the documentation?

File: spec/embedded_sass.proto [hunk 2]
```diff
@@ -257,6 +260,9 @@ message InboundMessage {
       // An error message explaining why the URL could not be loaded.
       string error = 3;
     }
+
+    // Whether `containing_url` in `FileImportRequest` is unused.
+    bool containing_url_unused= 4;
```
Comment (nex3) [inline spec/embedded_sass.proto]: Style nit:
```suggestion
    bool containing_url_unused = 4;
```

**Review by nex3 — CHANGES_REQUESTED** [review]

File: spec/embedded_sass.proto (hunk 1 above)
Comment (ntkme) [reply spec/embedded_sass.proto:203]: Added. Please feel free to edit this PR if you would like to rephrase the language.

**Review by nex3 — APPROVED** [review]

**Pull request merged**